test/auto/tests/tests.xml

```diff
@@ -36,6 +36,7 @@
         <step>/opt/tests/libqofono-qt5/runtest.sh tst_qofonocellbroadcast</step>
       </case>
       <case name="tst_qofonoconnmancontext">
+        <!-- EXPECTFAIL: Not supported by Sailfish ofono -->
```

Comment:
If not supported, should we comment out the whole test?

Reply:
Maybe, but I was thinking that it's just a build flag, so it could become supported again and then it would be nice to know.

Apparently the other tests marked as EXPECTFAIL in similar fashion are set as manual tests so they are not run in QA. I don't think that's a good way to skip them though.

Reply:
Could make sense to keep these consistent so if the other expected failures are skipped on the QA, then also skipping this with the same mechanism would be at least that.

Then again if failure is really expected, then failure could be considered as success and vice versa? So something switching the return values between zero and non-zero. That would make a difference on the results if ofono side ever gets changed.